packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -226,3 +222,13 @@ module.exports = {
     tls: 'empty'
   }
 };
+
+// @remove-on-eject-begin
+// Override babel-loader options to not read .babelrc.
+var babelLoader = module.exports.module.loaders.find(l => l.loader === 'babel');
+babelLoader.query = babelLoader.query || {};
+babelLoader.query.babelrc = false;
+// We embed the preset instead of referring to it because otherwise
+// `env` options in the preset do not get taken into account.
```

Comment:
Created #720. I can take a look at it at some point. Another peculiarity that we have is the use of `require.resolve()` instead of `require()` as the presets usually do. (Not sure if it's related to the issue though.) I quickly tried changing the preset to use `require()`, but then some plugins become undefined.

Anyway, happy to get this shipped as is, it's a good improvement to the error messages.